README-de.md

```diff
@@ -0,0 +1,111 @@
+# react-howto
+
+Wenn man mit React, oder Frontendentwicklung allgemein, beginnt, kann das Ökosystem verwirrend sein. Dafür gibt es mehrere Gründe:
+
+* React war ursprünglich an Early Adopter und Experten gerichtet
+* Facebook veröffentlicht nur Open-Source Software, die es auch selbst nutzt. Deshalb sind alle Tools um diese Software nicht für Projekte ausgelegt, die kleiner als Facebook sind.
+* Es gibt viel Marketing, das sich als React Einführung verkleidet
+
+Dieses Dokument setzt voraus, dass der Leser bereits eine Webseite mit HTML, CSS und JavaScript geschrieben hat.
+
+## Warum auf mich hören?
+
+Es gibt jede Menge widersprüchliche Empfehlungen und Best Practices zu React da draußen - warum solltest du auf mich hören?
+
+Ich war Mitglied des ursprünglichen Teams bei Facebook, das React gebaut und als Open-Source veröffentlicht hat. Mittlerweile arbeite ich nicht mehr bei Facebook sondern bei einem kleinen Startup, wodurch ich auch die Perspektive von außerhalb Facebooks kenne.
+
+## Wie man ins React Ökosystem reinkommt
+
+Alle Software baut auf einem bestimmten Technologie-Stack auf, den man hinreichend gut genug beherrschen muss, um seine Anwendung bauen zu können. Der Grund, warum das Ökosystem von React-Tools leicht überwältigend wirkt, ist, dass es bislang in der falschen Reihenfolge erklärt wurde.
+
+Du solltest React und seine Tools in folgender Reihenfolge lernen - **ohne nach vorne zu springen oder mehrere der Teile gleichzeitig zu lernen**:
+
+* [React selbst](#react-selbst-lernen)
+* [`npm`](#npm-lernen)
+* [JavaScript „Bundler“](#javascript-bundler-benutzen-lernen)
+* [ES6](#es6-lernen)
+* [Routing](#routing-lernen)
+* [Flux](#flux-lernen)
+
+**Du musst nicht alle Themenblöcke lernen um React produktiv einsetzen zu können.** Schau dir den nächsten Themenblock erst an, wenn du im aktuellen Probleme hast, die du mit den bestehenden Mitteln nicht lösen kannst.
+
+Des weiteren gibt es ein paar Themen, die in der React-Community viel diskutiert werden, die „bleeding edge“ sind. Die folgenden Themen sind interessant, aber schwer zu verstehen und viel weniger weitverbreitet als die eben genannten. **Für die meisten Anwendungen sind sie nicht notwendig.**
```

Comment:
too many adjectives and adverbs? I'd say "... aber schwer zu verstehen und weniger verbreitet als ..."